diff --git a/config/cc/gcc.in b/config/cc/gcc.in
index 5051804..bd8bcb3 100644
--- a/config/cc/gcc.in
+++ b/config/cc/gcc.in
@@ -36,6 +36,11 @@ choice
 # Don't remove next line
 # CT_INSERT_VERSION_BELOW
 
+config CC_V_4_9_2
+    bool
+    prompt "4.9.2"
+    select CC_GCC_4_9
+
 config CC_V_4_9_1
     bool
     prompt "4.9.1"
@@ -492,6 +497,7 @@ config CC_VERSION
     string
 # Don't remove next line
 # CT_INSERT_VERSION_STRING_BELOW
+    default "4.9.2" if CC_V_4_9_2
     default "4.9.1" if CC_V_4_9_1
     default "4.9.0" if CC_V_4_9_0
     default "linaro-4.8-2014.01" if CC_V_linaro_4_8
